usecases/objects/fakes_for_test.go

```diff
@@ -142,6 +142,15 @@ func (f *fakeVectorRepo) Exists(ctx context.Context,
 	return args.Bool(0), args.Error(1)
 }
 
+func (f *fakeVectorRepo) Object(ctx context.Context, cls string,
```

Comment:
For the maintainability purposes I would change the argument name from `cls` to `className` just to know quickly with a glance what type of input represents this argument

Reply:
This is just an internal function in a test file.

Reply:
Yes, it is, but nonetheless `class` argument can have a nice name :)

Reply:
This is just an internal function in a test file.